src/goto-programs/validate_goto_model.cpp

```diff
@@ -0,0 +1,251 @@
+/*******************************************************************\
+
+Module: Validate Goto Programs
+
+Author: Diffblue
+
+Date: Oct 2018
+
+\*******************************************************************/
+#include "validate_goto_model.h"
```

Comment:
Nit pick: add a blank line before this one.

Reply:
Done.